ut/columns_ut.cpp

```diff
@@ -114,6 +152,135 @@ TEST(ColumnsCase, DateAppend) {
     ASSERT_EQ(col2->At(0), (now / 86400) * 86400);
 }
 
+TEST(ColumnsCase, DateTime64_0) {
+    auto column = std::make_shared<ColumnDateTime64>(0ul);
+
+    ASSERT_EQ(Type::DateTime64, column->Type()->GetCode());
+    ASSERT_EQ("DateTime64(0)", column->Type()->GetName());
+    ASSERT_EQ(0u, column->GetPrecision());
+    ASSERT_EQ(0u, column->Size());
+}
+
+TEST(ColumnsCase, DateTime64_6) {
+    auto column = std::make_shared<ColumnDateTime64>(6ul);
+
+    ASSERT_EQ(Type::DateTime64, column->Type()->GetCode());
+    ASSERT_EQ("DateTime64(6)", column->Type()->GetName());
+    ASSERT_EQ(6u, column->GetPrecision());
+    ASSERT_EQ(0u, column->Size());
+}
+
+TEST(ColumnsCase, DateTime64_Append_At) {
+    auto column = std::make_shared<ColumnDateTime64>(6ul);
+
+    const auto data = MakeDateTime64s();
+    for (const auto & v : data) {
+        column->Append(v);
+    }
+
+    ASSERT_EQ(data.size(), column->Size());
+    for (size_t i = 0; i < data.size(); ++i) {
+        ASSERT_EQ(data[i], column->At(i));
+    }
+}
+
+TEST(ColumnsCase, DateTime64_Clear) {
+    auto column = std::make_shared<ColumnDateTime64>(6ul);
```

Comment:
It is worth adding `ASSERT_EQ(0u, column->Size());` to ensure that a column is created empty.

Reply:
We check that in other tests.